pkg/ring/kv/multi.go

```diff
@@ -0,0 +1,357 @@
+package kv
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/go-kit/kit/log/level"
+	"github.com/uber-go/atomic"
+
+	"github.com/cortexproject/cortex/pkg/util"
+)
+
+var (
+	primaryStoreGauge = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "consul_multikv_primary_store",
+		Help: "Selected primary KV store",
+	}, []string{"store"})
+
+	mirrorEnabled = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
`primaryStoreGauge` ends with `Gauge`. I personally find it more clear when reading the code if you add `Gauge` and `Counter` suffix to the other metrics too.